app/web/.prettierignore

```diff
@@ -2,3 +2,9 @@ proto/
 node_modules
 **/locales/*.json
 .next/
+coverage/
+junit.xml
+dl-protos.sh
+*.Dockerfile
+*.log
+yarn.lock
```

Comment:
```suggestion
```
We should keep the lock file committed like before!

Reply:
These are all committed, but this just stops prettier from looking at them. It seems when it doesn't find a formatter like for this extension, it'll return an error code

Reply:
Ahh yeah my bad! Misread that file as `gitignore`